Makefile

```
@@ -25,8 +25,17 @@ app-security-check: ## to check if any security issues in the PHP dependencies
 app-static-analysis: ## to run static analysis
 	php -dmemory_limit=-1 vendor/bin/phpstan analyze src tests -l 7
 
+files := $(wildcard snippets/*)
 app-test: ## to run unit tests
 	vendor/bin/phpunit
+	@for filename in $(files); do \
```

```suggestion
	@for filename in $$(find snippets -type f -name *.php); do \
```
so no need anymore `files`